SemiTruck.cpp

```diff
@@ -0,0 +1,28 @@
+#include "SemiTruck.h"
+
+SemiTruck::SemiTruck(const std::string& n) : Vehicle(n) {}
+
+void SemiTruck::rightLane( )
+{
+    setSpeed(65);
+    std::cout << "Semi: Stay in the right lane" << std::endl;
+}
+
+void SemiTruck::waySide()
```

Comment:
Name functions based on what they do.
Neither of these function names include a verb.
- [ ] revise